### PR TITLE
Refine builds and test

### PR DESCRIPTION
**Pull request by JanuszL**
Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

### CODE REVIEW

**Comment by JanuszL** [comment]
Builds 611190

File: qa/L3_RN50_convergence/test_pytorch.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+TOP1=$(grep "^##Top-1" $LOG | awk '{print $2}')
+TOP5=$(grep "^##Top-5" $LOG | awk '{print $2}')
+
+if [[ -z "$TOP1" || -z "$TOP5" ]]; then
+    echo "Incomplete output."
+    CLEAN_AND_EXIT 3
+fi
+
+TOP1_RESULT=$(echo "$TOP1 $MIN_TOP1" | awk '{if ($1>$2) {print "OK"} else { print "FAIL" }}')
```
Comment (Kh4L) [inline qa/L3_RN50_convergence/test_pytorch.sh]: Maybe it should be `>=` here (even if the equality would hardly occur)

Comment (JanuszL) [reply qa/L3_RN50_convergence/test_pytorch.sh]: Done

File: docs/examples/color_space_conversion.ipynb [hunk 2]
```diff
@@ -18,10 +18,24 @@
...
-   "execution_count": 4,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/usr/local/lib/python2.7/dist-packages/nvidia/dali/backend.py:15: FutureWarning: pybind11-bound class 'nvidia.dali.backend_impl.TensorCPU' is using an old-style placement-new '__init__' which has been deprecated. See the upgrade guide in pybind11's docs. This message is only visible when compiled in debug mode.\n",
```
Comment (Kh4L) [inline docs/examples/color_space_conversion.ipynb]: Please remove these messages: they are printed when `backend_impl` is compiled in Debug mode

Comment (Kh4L) [reply docs/examples/color_space_conversion.ipynb]: (I think opening the `.ipynb` in a text editor and removing `"outputs"` manually will do it here)

Comment (JanuszL) [reply docs/examples/color_space_conversion.ipynb]: Debug build. I will clean that up.

Comment (JanuszL) [reply docs/examples/color_space_conversion.ipynb]: Done

**Comment by JanuszL** [comment]
Finally builds 611431

**Pull request merged**